pennylane/tape/measure.py

```diff
@@ -386,3 +386,54 @@ def circuit():
     """
     # pylint: disable=protected-access
     return MeasurementProcess(State)
+
+
+def density_matrix(wires):
+    r"""Quantum density matrix in the computational basis.
+
+    .. note::
+
+        The density matrix can only be returned in tape mode:
+
+        >>> qml.enable_tape()
+
+        For more details on tape mode, see :mod:`pennylane.tape`.
+
+    This function accepts no observables and instead instructs the QNode to return its density
+    matrix (respectively reduced density matrix). The ``wires`` argument gives the possibility
```

Comment:
```suggestion
    matrix or reduced density matrix. The ``wires`` argument gives the possibility
```
The "respectively" part was not clear to me.